solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -19,7 +27,7 @@ git clone https://github.com/Seagate/cortx-re && cd $PWD/cortx-re/solutions/comm
 ```
 aws sts get-caller-identity
 ```
-- Modify `user.tfvars` file on local host with your AWS details.
```

Comment:
Can we keep `on local host` to be more specific

Reply:
fixed